### PR TITLE
WebSocket endpoint

### LINKED ISSUE #111: feat: WS endpoint for loxsi
Add an endpoint where loxsi can open a websocket to. We should be able to send data to out from other classes (singleton pattern?)

### PR DESCRIPTION
**Pull request by jnms-me**
Infrastructure for sending events to a loxsi over incoming WebSocket.
Only a single example event in a bad format is sent over it right now.
Closes #111

### CODE REVIEW

File: src/main/java/telraam/api/TeamResource.java
Comment (Topvennie) [inline src/main/java/telraam/api/TeamResource.java:1]: Is a nice potential feature.
However the websocket will only be used to send data about a team's position.
Loxsi still uses api calls to fetch the data about teams, laps, ...

**Comment by Topvennie** [comment]
Looks good thanks!

File: src/main/java/telraam/api/TeamResource.java
Comment (Topvennie) [reply src/main/java/telraam/api/TeamResource.java:1]: [5d42686](https://github.com/12urenloop/Telraam/pull/119/commits/5d42686d1b858a6371ca3f540e50472065028106)

**Comment by FKD13** [comment]
Thanks!

**Review by FKD13 — APPROVED** [review]

**Pull request merged**